### PR TITLE
feat(api-mock): enable dynamic reset of consumer offsets

### PR DESCRIPTION
**Pull request by craicoverflow**
This PR does two things:

- You can pre-seed the mock server by passing `--pre-seed`
- Updated the `resetConsumerGroupOffset` handler to give the ability to dynamically reset offset based on parameters.

### CODE REVIEW

**Comment by wtrocki** [comment]
I somehow missed that PR. Looking into this now

File: packages/api-mock/README.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ npm install -g @rhoas/api-mock
 asapi
 ```
 
+Optionally pre-seed with data:
```
Comment (wtrocki) [inline packages/api-mock/README.md:34]:
```suggestion
Starting mock with sample data present:
```

File: packages/api-mock/src/index.js [hunk 2]
```diff
@@ -36,7 +42,7 @@ const srsDataApi = new OpenAPIBackend({
 });
 
 // register handlers
-kafkaAPI.register(kafkaHandlers);
+kafkaAPI.register(createKafkaHandlers(preSeed));
 topicAPI.register(topicHandlers);
 srsControlApi.register(srsHandlers);
```
Comment (wtrocki) [inline packages/api-mock/src/index.js:47]: We need to have the same for srsHandlers for control plane. Let me know if you have time to apply this if not I can do it.

**Comment by wtrocki** [comment]
@rkpattnaik780 Do you mind verifying reset offset with kafka-ui? - I have checked it with the swagger UI and reviewed this PR.

**Comment by wtrocki** [comment]
Amazing work @craicoverflow Thank you for adding yargs support. It will be very easy now to add some extra ports etc.

File: packages/api-mock/src/index.js (hunk 2 above)
Comment (craicoverflow) [reply packages/api-mock/src/index.js:47]: I'll do it.

**Comment by craicoverflow** [comment]
@wtrocki I have added seed data for service registry!

**Review by wtrocki — APPROVED** [review]

**Pull request merged**